plugins/com.google.cloud.tools.eclipse.appengine.login/META-INF/MANIFEST.MF

```diff
@@ -19,3 +19,6 @@ Bundle-Classpath: lib/google-api-client-1.22.0.jar,
  lib/google-oauth-client-1.22.0.jar,
  lib/jackson-core-2.1.3.jar,
  .
+Export-Package: com.google.api.client.auth.oauth2;x-friends:="com.google.cloud.tools.eclipse.appengine.deploy",
+ com.google.cloud.tools.eclipse.appengine.login.ui;x-friends:="com.google.cloud.tools.eclipse.appengine.deploy"
```

Comment:
Question: now I see 4 instances of `x-friends` (including these two) across all the bundles. Should we always use this directive?